lib/modules/datasource/repology/readme.md

```diff
@@ -50,4 +50,4 @@ When the operating system package for `gcc` of `Alpine Linux 3.12` is updated, R
 <!-- prettier-ignore -->
```

Comment:
I'd also search for just the word `semantic` to find other mentions of the concept.

Reply:
I have searched for semantic, but couldn't find anything relevant.